Source/HoudiniEngineRuntime/Private/HoudiniEngine.cpp

```diff
@@ -155,19 +155,35 @@ FHoudiniEngine::StartupModule()
 			);
 	}
 
-	// Create static mesh Houdini logo.
-	HoudiniLogoStaticMesh =
-		LoadObject<UStaticMesh>(nullptr, HAPI_UNREAL_RESOURCE_HOUDINI_LOGO, nullptr, LOAD_None, nullptr);
-	HoudiniLogoStaticMesh->AddToRoot();
-
-	// Create default material.
-	HoudiniDefaultMaterial =
-		LoadObject<UMaterial>(nullptr, HAPI_UNREAL_RESOURCE_HOUDINI_MATERIAL, nullptr, LOAD_None, nullptr);
-	HoudiniDefaultMaterial->AddToRoot();
+    // [PHX][jsuter] - BEGIN modification        
```

Comment:
You can ignore this part of the pull request. The problems with houdini_default_material.uasset and houdini_logo.houdini_log are on our end and not yours. So this is just a temporary local work-around.